sdk/core/azure-core/CMakeLists.txt

```diff
@@ -179,6 +179,9 @@ endif()
 
 if(BUILD_TRANSPORT_CURL)
   target_link_libraries(azure-core PUBLIC CURL::libcurl)
+  if(WIN32)
+    target_link_libraries(azure-core PRIVATE Ws2_32)
```

Comment:
@BillyONeal do we need to declare this as a dependency in our vcpkg.json file, like we do `wil`?
https://github.com/Azure/azure-sdk-for-cpp/blob/f587bf9e797c8878436a8cacba3781e99eb2ae31/vcpkg.json#L22-L23
And add a `find_dependency` call here:
https://github.com/Azure/azure-sdk-for-cpp/blob/53c9da155244b72ac90ad27f515d4ea1e7505a01/sdk/core/azure-core/vcpkg/Config.cmake.in#L14

Reply:
@ahsonkhan QQ: What is the vcpkg package containing the WIn32 winsock headers?

As far as I know, there isn't one - that lib comes from the windows SDK. And that's not a vcpkg package. 

wil is a vcpkg package so we can (and must) take a dependency on it.

Reply:
What @LarryOsterman said. That lib comes from the Windows SDK. find_dependency goes with find_package which isn't used here.